src/Stratis.Features.FederatedPeg/FederatedPegSettings.cs

```diff
@@ -135,7 +112,7 @@ public FederatedPegSettings(NodeSettings nodeSettings, CounterChainNetworkWrappe
             this.FederationNodeIpAddresses = new HashSet<IPAddress>(endPoints.Select(x => x.Address), new IPAddressComparer());
 
             // These values are only configurable for tests at the moment. Fed members on live networks shouldn't play with them.
-            this.CounterChainDepositStartBlock = configReader.GetOrDefault(CounterChainDepositBlock, this.IsMainChain ? 1 : StraxMainDepositStartBlock);
+            this.CounterChainDepositStartBlock = configReader.GetOrDefault(CounterChainDepositBlock, 1);
```

Comment:
Realistically this won't be before the POW era has ended, so we can make this 12500?

Reply:
Let's do this in the next PR. Another change will be another hour before CI completes.